layers/+web-services/search-engine/packages.el

```diff
@@ -73,6 +73,7 @@
               (wolfram-alpha
                :name "Wolfram Alpha"
                :url "https://www.wolframalpha.com/input/?i=%s")))
+      (setq search-engine-alist (append search-engine-alist search-engine-config-list))
```

Comment:
Nice fix, however you should not invoke `setq` a second time. Rather use the backtick notation and add the content of `search-engine-config-list` directly at creation time to `search-engine-alist`

Here is a minimal example how to do this:
```elisp
  (setq mmm '(1 2 3))
  (setq myList
        `(Bla ;;; Note the backtick ` this is no quote '
          bla
          ,@mmm)) ;;; values starting with , will be evaluated. @ means the content of the list shall be inserted here.
```

Reply:
If this is a new concept, then the Emacs manuals explanation might also be helpful:
https://www.gnu.org/software/emacs/manual/html_node/elisp/Backquote.html